src/web/modules/agreements/list.njk

```diff
@@ -0,0 +1,130 @@
+    <div class="govuk-body">
+      <h4 class="govuk-heading-s">Filtered by</h4>
+      {% if filters.reference %}
+        <div class="govuk-body">
+          <div class="govuk-grid-column-one-third">Reference: </div>
+          <span><strong class="govuk-tag">{{ filters.reference }}</strong></span>
+        </div>
+      {% endif %}
+      {% if filters.email %}
+        <div class="govuk-body">
+          <div class="govuk-grid-column-one-third">Email:</div>
+          <span><strong class="govuk-tag">{{ filters.email }}</strong></span>
+        </div>
+      {% endif %}
+    </div>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  {% endif %}
+
+  {% if filters.gateway_agreement_id %}
+  <div class="govuk-body">
+    <span>Filtered by gateway agreement ID </span>
+    <span><strong class="govuk-tag">{{ filters.gateway_agreement_id }}</strong></span>
+  </div>
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  {% endif %}
+
+
+  <div>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Reference</th>
+          <th class="govuk-table__header" scope="col"></th>
+          <th class="govuk-table__header" scope="col">Description</th>
+          <th class="govuk-table__header" scope="col">Date created</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        {% for agreement in agreements %}
+          <tr class="govuk-table__row" onclick="local.href='/agreements'">
+            <td class="govuk-table__cell">
+              <a class="govuk-link govuk-link--no-visited-state" href="/agreements/{{ agreement.external_id }}">
+                {{ agreement.reference | truncate(20) }}
+              </a>
+            <td class="govuk-table__cell">
+              {{ status(
+                agreement.status
+                ) }}
+            </td>
+            <td class="govuk-table__cell">
+              <span class="govuk-caption-m">{{ agreement.description | truncate(20) }}</span>
+            </td>
+            <td class="govuk-table__cell">
+              <span class="govuk-caption-m">{{ agreement.created_date | formatDateLocalTimeZone }}</span>
+            </td>
+          </tr>
+        {% else %}
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell center" colspan="5"><span><i>No agreements found.</i></span></td>
+        </tr>
+        {% endfor %}
+      </tbody>
+    </table>
+  </div>
+
+  <div class="govuk-body">
+    Showing page {{ set.page }} ({{ set.count }}) of
+    {% if set.total < 5000 %}
+        about {{ set.total }} agreements
+    {% else %}
+        over 5000 agreements
+    {% endif %}
+  </div>
+  <div class="toolbox-right">
+    <a
+      {% if set.page <= 1 %}
+        disabled
+      {% elseif filters.gateway_payout_id %}
```

Comment:
Similarly payout filtering logic can be removed from this link and the link below

Reply:
Thanks